### PR TITLE
[Deprecation] Deprecate master_timeout in favor of cluster_manager_timeout

### PR DESCRIPTION
**Pull request by manasvinibs**
Signed-off-by: manasvinibs <manasvis@amazon.com>

### Description
As part of the meta issue opensearch-project/OpenSearch#2589 to track the plan and progress of applying inclusive naming across OpenSearch Repositories.

 
In this change we are generating the json files for the _cat API 's specs by running following sample cmd:
```node scripts/spec_to_console.js -g "../OpenSearch/rest-api-spec/src/main/resources/rest-api-spec/api/cat.cluster_manager.json" -d "src/plugins/console/server/lib/spec_definitions/json/generated"```

### Issues Resolved
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1685
 
### Check List
- [ ] New functionality includes testing.
  - [X] All tests pass
    - [ ] `yarn test:jest`
    - [ ] `yarn test:jest_integration`
    - [ ] `yarn test:ftr`
- [ ] New functionality has been documented.
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
       "bytes": [
         "b",
         "k",
-        "osd",
+        "kb",
```
Comment (joshuarrrr) [inline src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json:8]: My guess is that these are regressions? Naively, I'd think `osd` is the correct value.

File: src/plugins/console/server/lib/spec_definitions/json/generated/indices.create.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
   "indices.create": {
     "url_params": {
-      "include_type_name": "__flag__",
```
Comment (joshuarrrr) [inline src/plugins/console/server/lib/spec_definitions/json/generated/indices.create.json:4]: this change may be fine, but seems unrelated to the other changes.

**Review by joshuarrrr — COMMENTED** [review]
Looking good! Two things for further investigation:

1. I'm suspicious of changes from `"osd"` to `"kb"`
2. I want to make sure we understand/document the removal of `"include_type_name"` or `{type}` as a param. I'm guessing this was another change to the js client since the last time these were updated? But worth confirming and including in the commit info.

File: src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json (hunk 1 above)
Comment (manasvinibs) [reply src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json:8]: This got replaced on running the spec generation script with latest OpenSearch "main" branch. Is there a way I can verify the correct value?

File: src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
       "bytes": [
         "b",
         "k",
-        "osd",
```
Comment (kavilla) [inline src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json:8]: Ha, whoopsie.

File: src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json (hunk 1 above)
Comment (kavilla) [reply src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json:8]: `kb` is the right value. When we forked and did renaming we just went ahead and just renamed a lot of references. This `kb` is related to `bytes` so it we seem we had an existing bug.

**Comment by kavilla** [comment]
>  I want to make sure we understand/document the removal of "include_type_name" or {type} as a param. I'm guessing this was another change to the js client since the last time these were updated? But worth confirming and including in the commit info.

I think this is a good change though. `type` was removed for `2.x`

**Review by joshuarrrr — APPROVED** [review]
Sounds like all these changes are good - I'd just recommend also including the non-naming changes in the commit message just so it's obvious they were intentional.

**Review by kavilla — APPROVED** [review]
LGTM! Will mention what @joshuarrrr  recommended on merge.
